apps/site/src/pages/api/me/revoke-api-key.api.ts

```diff
@@ -0,0 +1,44 @@
+import { validationResult } from "express-validator";
+import { body as bodyValidator } from "express-validator/src/middlewares/validation-chain-builders";
+
+import { createAuthenticatedHandler } from "../../../lib/api/handler/authenticated-handler";
+import { ApiKey } from "../../../lib/api/model/api-key.model";
+import { formatErrors } from "../../../util/api";
+
+export type ApiRevokeApiKeyBody = {
+  publicId: string;
+};
+
+export type ApiRevokeApiKeyResponse = {};
+
+export default createAuthenticatedHandler<
+  ApiRevokeApiKeyBody,
+  ApiRevokeApiKeyResponse
+>(false)
```

Comment:
`false` means that users who have not completed signup can use this route. 

I see that I put this on the original API key handler which explains why it's here, but I was wrong. We can just omit it as it will default to `true` (check if the user has completed sign up).

```suggestion
>()
```

Reply:
Addressed in 6755f5529c5ed899bc23590559c28f50d5b3e573, also changed it for Create/List